.github/workflows/auto-format.yml

```
@@ -0,0 +1,49 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18.x
+          cache: "npm"
+      - name: Install Prettier
+        run: npm ci
+      - name: Auto-fix code formatting (Prettier)
+        run: npx prettier --write .
+
+      - name: Commit changes
+        id: commit
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:8:28: Double quote to prevent globbing and word splitting

(shellcheck)